_topic_maps/_topic_map.yml

```diff
@@ -3,6 +3,8 @@ Name: About OpenShift Builds
 Dir: builds
 Distros: openshift-builds
 Topics:
+- Name: Release Notes
```

Comment:
Should this title be more specific. Some other component group could also create a topic called "Release Notes" and processors might spew out errors.